docs/endpoint.md

```diff
@@ -5,14 +5,118 @@ The endpoint is one of the components provided by Tangle-accelerator, running on
 The encrypted message would be sent to Tangle with a streaming message channel API. The streaming message channel API would ensure the order of messages in the channel. The user who wants to fetch/send message to Tangle needs to provide `data_id`, `key` and `protocol` to identify a specific message.
 A message sent by endpoint needs to be encrypted locally, which avoids message being peeked and modified. 
 
-# How to use
+# How to build endpoint
+
+## Setup Sierra development environment 
+The endpoint uses the Sierra framework as based runtime system. Developers need to set up the Sierra development environment to build endpoint as specific target.
+
+### How to build endpoint application for wp77xx
+
+[Leaf](https://docs.legato.io/latest/toolsLeaf.html) is a workspace manager that will download, install and configure the required software packages for a Legato development environment.
+
 ```
-$ bazel build //endpoint:wp7702
-$ bazel build //endpoint:sim
+$ curl -sLO https://downloads.sierrawireless.com/tools/leaf/leaf_latest.deb && sudo dpkg -i leaf_latest.deb
```

Comment:
The dependent packages should be installed in advance, such as `python3-argcomplete` and/or `python3-gnupg`.